actions/image_partition_action.go

```diff
@@ -436,6 +445,14 @@ func (i ImagePartitionAction) Run(context *debos.DebosContext) error {
 			}
 		}
 
+		if p.PartType != "" {
+			err = debos.Command{}.Run("sfdisk", "sfdisk", "--part-type", context.Image, fmt.Sprintf("%d", p.number), p.PartType)
```

Comment:
can we do this as part of the parted call rather then needing another tools ?

Reply:
doesn't look like parted can set raw guids

Reply:
unfortunately some more peeking into parted, only handles setting basic guids: https://git.savannah.gnu.org/gitweb/?p=parted.git;a=blob;f=libparted/labels/gpt.c;h=93f7add2a2411f035f6c4991506871c3e1e25663;hb=HEAD#l1458